packages/runtime/src/parser/parser.ts

```diff
@@ -960,3 +960,30 @@ export function getProgramVersion(program: string): number {
 	opcodeFromSentence(wordsFromLine(firstLine), 1, ip, ExecutionMode.APPLICATION);
 	return ip.tealVersion;
 }
+
+/**
+ *
+ * @param str string to convert
+ * @returns camelCase to hypenated string eg(fooBar to foo-bar)
+ */
+export function convertCapitalToHyphens(str: string): string {
+	return str.replace(/[A-Z]/g, (m) => "-" + m.toLowerCase());
+}
+
+/**
+ *
+ * @param object
+ * @returns object keys parsed from camelCase to hypenated str eg(fooBar to foo-bar)
+ */
+export function convertKeysToHyphens(object: any) {
```

Comment:
Please avoid using `any`. You can use `Object`

Reply:
but then we get the type error below `object[key]`,and we are not sure about the type

Reply:
then how about using `unknown`?

Reply:
Argument of type 'unknown' is not assignable to parameter of type '{}'.
  i get this error then